user_manual/changes/V107-027.yaml

```diff
@@ -0,0 +1,6 @@
+type: bugfix
+title: Fix ``is_constant_object`` property
+description: |
+    This fix adds a missing case for the ``is_constant_object`` property where a
+    call expression refers to a subprogram without specification.
+date: 2022-01-07
```

Comment:
A change entry is not necessary: we didn't have any release since this property was added, so no customer could be able to witness this bug.